### ISSUE TITLE
Support for Styled Components 3.0.0+

### ISSUE THREAD

**Issue by jimmycann** [mbasso/styled-components-test-utils#5]
**General Information**

- [ ] Bug
- [ ] Improvement
- [ ] Feature
- [X] Other

**Description**

Add support for Styled Components V3 and up

Currently updating a projects' styled-components dependency above 3.0.0 results in the following error

```js
Error: Cannot find module 'styled-components/lib/models/StyleSheet'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
```

This is a result of SC only shipping flat bundles from V3, as outlined in their release notes: https://github.com/styled-components/styled-components/releases/tag/v3.0.1

Will submit a PR to add support shortly

**Steps to reproduce**

Update Styled Components to version 3.0.0 plus and run tests

**Versions**

- styled-components-test-utils: 0.3.1
- node: 8.10

- jest: N/A
- expect: N/A
- chai: 3.5.0
- jasmine: N/A

**Comment by mbasso** [comment]
Awesome!! 😄
Thank you so much for opening this issue and for following through with a PR!
All looks good to me, I'll merge it and publish the new version shortly!

**Comment by mbasso** [comment]
Just published [styled-components-test-utils@0.4.0](https://github.com/mbasso/styled-components-test-utils/releases/tag/0.4.0), thank you again for contributing! 🎉🎉🎉

**Comment by jimmycann** [comment]
Thanks so much for getting to this so quick, and thanks for building this in the first place! It's been super helpful!
